contributing.md

```diff
@@ -202,7 +202,8 @@ The “page” modules themselves contain introductory blurbs and examples for t
 You can run the docs app locally using:
 
 ```sh
-npm run docs
+npm run build
+npm run docs:neo
```

Comment:
This script has been removed now it should just be `npm run docs`

Reply:
changed